### DISCUSSION TITLE
Interactions and interactivity support for mobile?

### DISCUSSION THREAD

**Discussion by nalonau** [unoplatform/uno#10341] [Q&A]
Does anyone know if its possible to get:

    xmlns:Interactivity="using:Microsoft.Xaml.Interactivity" 
    xmlns:Interactions="using:Microsoft.Xaml.Interactions.Core"

working on android / mobile targets? As all my events route through commands to the ViewModel, when clicking on an item in a GridView ive needed to use the interactions:

                            <Interactivity:Interaction.Behaviors>
                                <Interactions:EventTriggerBehavior EventName="ItemClick" SourceObject="{Binding ElementName=SiteListView}">
                                    <Interactions:InvokeCommandAction Command="{Binding SelectSiteCommand}"></Interactions:InvokeCommandAction>
                                </Interactions:EventTriggerBehavior>
                            </Interactivity:Interaction.Behaviors>

which works great in windows but not so in mobile.

On mobile (tried on Android) it doesn't register the command in the view model when you (for example click an item in a listview or gridview).

**Comment by jeromelaban** [comment]
Thanks for the report. Can you provide a repro project of the issue?

**Reply by nalonau** [reply]
Can do, I'll upload one hopefully in the next couple of days.
